tests/fixtures/simple-failing-dockerfile/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM busybox:latest
+LABEL com.docker.compose.test_failing_image=true
```

Comment:
Please use the existing label (`com.docker.compose.test_image`) for this, so it will be cleaned up without you having to add any cleanup to each test.